lib/app/migrate.ts

```diff
@@ -0,0 +1,638 @@
+import { CLIAccount } from '@hubspot/local-dev-lib/types/Accounts';
```

Comment:
This is kind of a nit, but would you mind ordering these imports? Mixing all the third-party imports with the local imports makes it hard to figure out if there's already an existing import from any given path.